### ISSUE TITLE
Duplicate audio streams

### ISSUE THREAD

**Issue by 123jjck** [crunchy-labs/crunchy-cli#118]
**Describe the bug**
When I try to download anime, the program displays two identical audio tracks for each language, and tries to download them all

**To Reproduce**
Steps / command to reproduce the behavior:
```
.\crunchy-v3.0.0-dev.8_windows.exe archive -l ru-RU -l ja-JP -s ru-RU -s en-US -o "{title}.mkv" https://www.crunchyroll.com/ru/series/GRGG9798R/rezero--starting-life-in-another-world-[S1E1]
```

**Expected behavior**
A file with Russian and Japanese audio should be downloaded, without duplicates

**Screenshots**
![image](https://user-images.githubusercontent.com/48408304/213216526-add817df-9831-4fa3-b185-c2cb42d51d0a.png)
![image](https://user-images.githubusercontent.com/48408304/213216753-472053e7-68f2-4c79-af82-01a33e83e027.png)

**Client**
 - OS: Windows 11 22H2
 - Version: [v3.0.0-dev.8](https://github.com/crunchy-labs/crunchy-cli/releases/tag/v3.0.0-dev.8)

**Comment by 123jjck** [comment]
For example i'm downloaded first episode of Re:ZERO in 240p and here are the video tracks displayed in MPC-HC:
![image](https://user-images.githubusercontent.com/48408304/213222950-76105df4-31bb-40ab-a505-153c42ccd3c2.png)

**Comment by 123jjck** [comment]
Also noticed that the second downloaded track of each language has a slightly smaller size:
![image](https://user-images.githubusercontent.com/48408304/213223196-6460efc8-0912-4536-920d-432855e2ede7.png)

**Comment by ByteDream** [comment]
The problem is that Crunchyroll has different episodes for the first Re:Zero episode. In some language, the first episode is split in two ~24 minute parts, other languages are providing it as a long ~50 minute episode. This distracts the cli.

**Comment by 123jjck** [comment]
> The problem is that Crunchyroll has different episodes for the first Re:Zero episode. In some language, the first episode is split in two ~24 minute parts, other languages are providing it as a long ~50 minute episode. This distracts the cli.

Yes, if you choose English dub, then the first episode will be 50 minutes long, and if you choose Japanese or Russian, then it will be divided into two parts. The problem is that the bug occurs even when English is not selected. English subtitles were selected in my command, and even when I removed them, the cli still tried to download duplicated video tracks. The "-m audio" option didn't help either. Will it be fixed and is there a workaround now?

**Comment by ByteDream** [comment]
The is currently no workaround for it. I'll investigate this issue further when I have proper time for it.
